test/error-messages/invalid-arg-value.txt

```diff
@@ -0,0 +1,12 @@
+Invalid value
+
+sqrt :: Number -> Number
+        ^^^^^^
+          1
+
+At position 1:
+  Where: 1st Argument
+  Value: null
+  Types: Null
+
+Expected types: Number.
```

Comment:
Invalid value

    sqrt :: Number -> Number
            ^^^^^^
              1

    1)  null :: Null

    The value at position 1 is not a member of Number.